nf_core/create.py

```diff
@@ -446,6 +469,10 @@ def fix_linting(self):
         if not self.template_params["github_badges"] or not self.template_params["github"]:
             lint_config["readme"] = ["nextflow_badge"]
 
+        # If the pipeline is unbranded
+        if not self.template_params["branded"]:
+            lint_config["files_unchanged"].extend([".github/ISSUE_TEMPLATE/bug_report.yml"])
```

Comment:
If we can avoid just flat out ignoring this file, that would be nice. There is already a [function](https://github.com/nf-core/tools/blob/0912990a63ef29e44e07cc2ba6ab81113684e0ae/nf_core/create.py#L371-L387) to customise it to remove some nf-core specific stuf. I think the failure on this file is something to do with the custom prefix / name match at the end of the output.

Reply:
When the pipeline is unbranded we remove one chunk of this file containing links to nf-core documentation:
```yaml
  - type: markdown
    attributes:
      value: |
        Before you post this issue, please check the documentation:

        - [nf-core website: troubleshooting](https://nf-co.re/usage/troubleshooting)
        - [{{ name }} pipeline documentation](https://nf-co.re/{{ short_name }}/usage)
```
so it won't match the template, and can't be considered a partial match either.

Reply:
> so it won't match the template, and can't be considered a partial match either.

Seems like this isn't possible so will merge this in and we can follow up in a separate PR / issue if required.